Tasks/XamarinLicense/XamarinLicense.ps1

```diff
@@ -12,7 +12,7 @@ Write-Verbose "email = $email"
 Write-Verbose "activateAndroid = $activateAndroid"
 Write-Verbose "timeout = $timeout"
 
-$activateAndroidLicense = Convert-String $activateAndroid Boolean
+$activateAndroidLicense = [System.Convert]::ToBoolean($activateAndroid)
```

Comment:
I'd rather do:
$activateAndroidLicense = ($activateAndroid -eq 'true')

to avoid an exception in case the input isn't defined. I don't know if that input always existed from the beginning, so i'm not sure whether it's realistic to be empty.

Adding Yang to follow-up on this PR.